locales/en-US/app.l20n

```diff
@@ -66,7 +67,22 @@
 <feedbackCancelButton "Close">
 
 <feedbackUninstallTitle "Thank You!">
-<feedbackUninstallCopy "Your participation in Firefox Test Pilot means a lot! Please check out our other experiments, and stay tuned for more to come!">
+<feedbackUninstallCopy "Your participation in Firefox Test Pilot means
+                        a lot! Please check out our other experiments,
+                        and stay tuned for more to come!">
+
+<discussNotifyTitle "Just one second...">
+<discussNotifyMessage "<p>In the spirit of experimentation, we're using a forum service
+                          that requires an account different from the one you
+                          created for Test Pilot. We are working hard to change this
+                          and make the forum experience smoother.</p>
+                       <p>If you don't feel like creating another account, you can
```

Comment:
@meandavejustice change this to "If you don't want to create another account," and then merge away!